compositing-2/Overview.bs

```diff
@@ -709,25 +709,23 @@ If the 'knock-out' property is disabled, each element within the group is only c
 -->
 <h3 id="pagebackdrop">The Page Group</h3>
 
-The top level group is the page group. All other elements and groups are composited into this group.
-The page group is an <a href="#isolatedgroups">isolated group</a>.
 
-The page group is composited with a backdrop color of white with 100% opacity.<!--<br>
-The page group may be used as an element in another graphical composition.-->
-<!--
-<p>
-For example,
-this is an SVG file that contains a red object at 50% opacity. <br>
-The UA will composite the page group onto a white background with 100% opacity. <br>
-The results are as follows:
-<pre>
-co = RGB(255, 0, 0) * .5 + RGB(255, 255, 255) * 1 * (1 - .5)
-co = RGB(127, 0, 0) + RGB(127, 127, 127)
-co = RGB(255, 127, 127)
-</pre>
-which is the color value ultimately displayed by the UA.
-</p>
--->
+The <a href="#isolatedgroups">isolated group</a for the root element is the
+page group. All other elements and groups are composited into this group.
+The background of the root element (if specified) is painted into the page
+group, and any filter, clip-path, mask and and opacity is then applied, before
+compositing into the <a href="#rootgroup">root group</a>.
+
+<h3 id="rootgroup">The Root Group</h3>
+The root group is the final compositing output, and is is above the page
+group. Compositing of the page group into the root group is performed with the
+specified blend mode of the root element.
+
+<p class="note">Browsers often use an infinite white, 100% opaque root group,
+for final compositing of a web page, but this is not required. For example,
+a browser may blend pages with a group generated via an external texture
+in the containing operating system.
```

Comment:
This example (external texture from the OS) may confuse more than it clarifies.

A simpler example is an iframe or SVG in an img element, which have a transparent canvas.

Reply:
Done.